package.json

```diff
@@ -5,36 +5,43 @@
   "main": "app/server.ts",
   "directories": {},
   "dependencies": {
+    "@types/node-fetch": "^2.5.7",
     "body-parser": "^1.18.3",
-    "express": "~4.16.4",
+    "express": "~4.17.1",
     "express-mung": "^0.5.1",
-    "osprey": "^0.5.4",
-    "osprey-mock-service": "^0.4.1",
+    "node-fetch": "^2.6.1",
+    "osprey": "^1.0.0",
+    "osprey-mock-service": "^1.0.0",
     "require-all": "^3.0.0",
     "ts-node": "^9.0.0",
-    "typescript": "^4.0.3",
-    "uuid": "^3.3.2"
+    "typescript": "^4.0.5",
+    "uuid": "^8.3.1"
   },
   "devDependencies": {
     "@semantic-release/commit-analyzer": "^8.0.1",
```

Comment:
Can you put a `"engines": {"node": ">=10.18.0"}` somewhere in the package.json as semantic-release requires a node version of >= 10.18.0. This allows for early error when installing node modules

[https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines)

Reply:
May also want to consider adding in `.nvmrc` to the root of the project with `v10.18.0` in the file for the people which use nvm for node version management https://github.com/nvm-sh/nvm

Reply:
Added.